### PR TITLE
docs(carousel/color-picker/color-select-panel/config-provider/crop/divider/link-menu/slide-bar/toggle-menu/user-contact/user-head/):update text correction description

### PR DESCRIPTION
**Pull request by James-9696**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: examples/sites/demos/pc/app/divider/webdoc/divider.js [hunk 1]
```diff
@@ -40,20 +40,20 @@ export default {
...
       'name': 'divider',
       'type': 'component',
-      'properties': [
+      'props': [
         {
           'name': 'color',
           'type': 'string',
           'defaultValue': '',
-          'desc': { 'zh-CN': '设置分隔线的颜色', 'en-US': 'Set the color of the divider.' },
+          'desc': { 'zh-CN': '设置分隔线的颜色。', 'en-US': 'Set the color of the divider.' },
```
Comment (zzcr) [inline examples/sites/demos/pc/app/divider/webdoc/divider.js]: api 属性里面的描述不用末尾句号

Comment (James-9696) [reply examples/sites/demos/pc/app/divider/webdoc/divider.js]: 已修复

File: examples/sites/demos/pc/app/color-picker/webdoc/color-picker.js [hunk 2]
```diff
@@ -5,79 +5,91 @@ export default {
...
+        'en-US': 'For details, see the following example.'
+      },
       'codeFiles': ['size.vue']
     },
     {
       'demoId': 'event',
       'name': { 'zh-CN': '事件触发', 'en-US': 'event' },
-      'desc': { 'zh-CN': '点击确认是将会触发confirm事件, 取消时触发cancel事件', 'en-US': 'When click confirm will trigger confirm event. When click outside or cancel will trigger cancel event' },
+      'desc': {
+        'zh-CN': '点击确认是将会触发confirm事件, 取消时触发cancel事件。',
```
Comment (chenxi-20) [inline examples/sites/demos/pc/app/color-picker/webdoc/color-picker.js]: 中英文之间需要用空格隔开，code标签高亮显示。
比如：点击确认是将会触发confirm事件
添加空格后：点击确认是将会触发 <code>confirm</code> 事件
具体可以查看一下整改好的组件文档~

File: examples/sites/demos/pc/app/carousel/webdoc/carousel.js [hunk 3]
```diff
@@ -122,80 +122,87 @@ export default {
...
+          'defaultValue': `该属性的默认值为 'hover'`,
+          'desc': { 'zh-CN': '切换箭头的显示时机;该属性的可选值为 always / hover / never。', 'en-US': '' },
           'demoId': 'up-down-carousel'
         },
         {
           'name': 'autoplay',
           'type': 'boolean',
           'defaultValue': '该属性的默认值为 false',
-          'desc': { 'zh-CN': '是否自动切换', 'en-US': '' },
+          'desc': { 'zh-CN': '是否自动切换。', 'en-US': '' },
```
Comment (yoyo201626) [inline examples/sites/demos/pc/app/carousel/webdoc/carousel.js]: 英文最好也补上

File: examples/sites/demos/pc/app/carousel/webdoc/carousel.js [hunk 4]
```diff
@@ -83,19 +83,19 @@ export default {
     {
       'demoId': 'carousel-arrow-always',
       'name': { 'zh-CN': '总是显示切换箭头', 'en-US': 'Always show the switch arrow' },
-      'desc': { 'zh-CN': '详细用法参考如下示例', 'en-US': 'For details, see the following example.' },
+      'desc': { 'zh-CN': '详细用法参考如下示例。', 'en-US': 'For details, see the following example.' },
```
Comment (yoyo201626) [inline examples/sites/demos/pc/app/carousel/webdoc/carousel.js]: 这种有特点的示例可以尝试把描述写得详细点。比如：

> 通过 xxx 设置是否总是显示切换箭头

File: examples/sites/demos/pc/app/carousel/webdoc/carousel.js (hunk 3 above)
Comment (James-9696) [reply examples/sites/demos/pc/app/carousel/webdoc/carousel.js]: 已修复

File: examples/sites/demos/pc/app/color-picker/webdoc/color-picker.js (hunk 2 above)
Comment (James-9696) [reply examples/sites/demos/pc/app/color-picker/webdoc/color-picker.js]: 已修复

File: examples/sites/demos/pc/app/carousel/webdoc/carousel.js (hunk 4 above)
Comment (James-9696) [reply examples/sites/demos/pc/app/carousel/webdoc/carousel.js]: 已修复

File: examples/sites/demos/pc/app/slide-bar/webdoc/slide-bar.js [hunk 5]
```diff
@@ -36,58 +39,61 @@ export default {
...
+            'zh-CN': '鼠标滚轮上下滚动的时候，滚动的块数长度',
+            'en-US': 'Number of scrolling blocks when the mouse wheel is scrolled up or down'
           },
           'demoId': 'wheel-blocks'
         },
         {
           'name': 'tag',
           'type': 'string',
-          'defaultValue': '该属性的默认值为 ul',
+          'defaultValue': `该属性的默认值为 '<ul>'`,
```
Comment (zzcr) [inline examples/sites/demos/pc/app/slide-bar/webdoc/slide-bar.js]: 这里不需要改吧，就是ul

File: examples/sites/demos/pc/app/slide-bar/webdoc/slide-bar.js [hunk 6]
```diff
@@ -36,58 +39,61 @@ export default {
...
           'desc': { 'zh-CN': '设置组件的 Dom 标签', 'en-US': 'Setting the Dom Label of a Component' },
           'demoId': 'custom-tag'
         },
         {
           'name': 'sub-tag',
           'type': 'string',
-          'defaultValue': '该属性的默认值为 li',
+          'defaultValue': `该属性的默认值为 '<li>'`,
```
Comment (zzcr) [inline examples/sites/demos/pc/app/slide-bar/webdoc/slide-bar.js]: 这里不需要改吧，就是li

Comment (James-9696) [reply examples/sites/demos/pc/app/slide-bar/webdoc/slide-bar.js]: 已修复

File: examples/sites/demos/pc/app/slide-bar/webdoc/slide-bar.js (hunk 5 above)
Comment (James-9696) [reply examples/sites/demos/pc/app/slide-bar/webdoc/slide-bar.js]: 已修复

**Review by yoyo201626 — APPROVED** [review]

**Review by zzcr — APPROVED** [review]

**Pull request merged**